ibm/service/project/data_source_ibm_project_event_notification.go

```diff
@@ -21,22 +21,10 @@ func DataSourceIbmProjectEventNotification() *schema.Resource {
 		ReadContext: dataSourceIbmProjectEventNotificationRead,
 
 		Schema: map[string]*schema.Schema{
-			"project_id": &schema.Schema{
```

Comment:
the API spec (swagger.yaml) has changed, now the parameter is simply `id`, can we safely still call it `project_id` in the terraform bundle?

Reply:
cool, done! I've run tests locally and it works, I commit the changes

Reply:
I think generator also provides you annotation to name the attrribute with user defined value

Reply:
I don't know how this was generated (maybe @dvesperini knows), I thought we need the `project_id` since the project id is an input parameter for our APIs.
Anyway, I don't have enough skills on this subject, I made sure the automated tests work fine, and also building the provider locally and using terraform commands init/plan/apply/destroy I'm able to manage resources, so I don't really understand the problem here (our previous PR was merged, this PR is intended to align the terraform provider code to the latest Projects SDK changes).

Reply:
Yes we used the generator.
@hkantare can you merge this PR?